examples/data_preprocessing/mimic_dataset/prepare_mimic_cxr.ipynb

```diff
@@ -0,0 +1,1520 @@
+    "The model is finetuned using only two epochs, since more epochs would overfit the model. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch 0/1\n",
+      "----------\n",
+      "train Loss: 1.8165 Acc: 0.4488\n",
+      "predictions tensor([ 8,  8,  1, 13])\n",
+      "predictions tensor([8, 8, 1, 1])\n",
+      "predictions tensor([13,  8, 13,  1])\n",
+      "predictions tensor([13, 12])\n",
+      "val Loss: 2.7613 Acc: 0.0714\n",
+      "\n",
+      "Epoch 1/1\n",
+      "----------\n",
+      "train Loss: 1.1641 Acc: 0.6416\n",
+      "predictions tensor([3, 0, 5, 9])\n",
+      "predictions tensor([0, 0, 0, 0])\n",
+      "predictions tensor([ 9,  0,  9, 13])\n",
+      "predictions tensor([13, 13])\n",
+      "val Loss: 2.8807 Acc: 0.0714\n",
+      "\n",
+      "Best val Acc: 0.071429\n"
+     ]
+    }
+   ],
+   "source": [
+    "model = models.resnet50(pretrained=True)\n",
+    "num_ftrs = model.fc.in_features\n",
+    "# set output size to 14 (number of classes)\n",
+    "model.fc = nn.Linear(num_ftrs, 14)\n",
+    "model = model.to(device)\n",
+    "criterion = nn.CrossEntropyLoss()\n",
+    "optimizer = optim.Adam(model.parameters(), lr=0.0001)\n",
+    "step_lr_scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=7, gamma=0.1)\n",
+    "model = train_model(model, criterion, optimizer, step_lr_scheduler, num_epochs=2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, that we have trained our model, we will extract the features. \n",
+    "\n",
+    "In the next step, the last layer is removed from the model, so the output is then the second last layer with dimension 1x2048.\n",
+    "Then we let all out data run through the model. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "modules = list(model.children())[:-1]\n",
+    "model=torch.nn.Sequential(*modules)\n",
+    "for p in model.parameters():\n",
+    "    p.requires_grad = False\n",
+    "    \n",
+    "model.eval()\n",
+    "#apply modified resnet50 to data\n",
+    "dataloaders = DataLoader(mimicDataset(input_list[:n,:]), batch_size=n,num_workers=0)\n",
+    "    \n",
+    "data, labels = next(iter(dataloaders))\n",
+    "with torch.no_grad():\n",
+    "    features_var = model(data)\n",
+    "    features = features_var.data \n",
+    "    all_X_finetuned = features.reshape(n,2048).numpy()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['all_X_finetuned.lib']"
+      ]
+     },
+     "execution_count": 35,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#save feature matrix\n",
```

Comment:
this comment could be a markdown text and then the `dump` code to the separate cell in order to highlight it

(also: mb I just overlooked it, but - have you saved the reports as well?)

Reply:
I have the reports in one column of the 'input_list_full' but no file with only the reports.